docs/visual-testing/integrations/csharp.md

```diff
@@ -335,6 +336,33 @@ VisualClient = VisualClient.Create(Driver, Region.UsWest1, sauceUsername, sauceA
 VisualClient.CaptureDom = true;
 ```
 
+### Full page screenshots
+
+<FullPageDescription />
```

Comment:
Not sure that's ideal:
- Casing is not the same
- Method names are not identical (`SauceVisualCheck` vs. `VisualCheck`)
- Not mentioning the `FullPageConfig` kind.

My first feeling when reading is that example and description does not match (which is the case actually).

Reply:
fixed